idle-rpg/game/utils/Event.js

```diff
@@ -140,6 +146,10 @@ class Event {
                 const eventLog = `Killed ${randomPlayer.name} in ${selectedPlayer.map.name}.`;
                 const otherPlayerLog = `Died to ${selectedPlayer.name} in ${selectedPlayer.map.name}.`;
 
+                if (randomPlayer.horse = 'Yes'){
+                  const eventLog = `Killed ${randomPlayer.name} in ${selectedPlayer.map.name}. ${randomPlayer.name}'s horse died in the fight.`;
+                }
```

Comment:
eventLog has to be a let above and just change the variable here or possibly just .concat() the horse string
also have to update the other players log